lib/generators/sufia/work/templates/locale.en.yml.erb

```diff
@@ -1,4 +1,10 @@
 en:
+  sufia:
+    select_type:
+      <%= file_name %>:
+        name:               "<%= human_name.titleize %>"
+        description:        "<%= human_name %> works"
```

Comment:
The idea is that people will change it.  It only really looks awful with `GenericWork`, but I assume most people will generate more specific work types and certainly will change these default values. That's why we're generating them right into the host app.